binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -241,7 +242,9 @@ public void initialize() {
             }
             onUpdate();
         } else {
-            logger.warn("Cannot connect to hue bridge. IP address or user name not set.");
+            String warn = "Cannot connect to hue bridge. IP address or user name not set.";
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR, warn);
+            logger.warn(warn);
```

Comment:
> Hm, if I configure my logger to log all messages of the Hue binding to a separate file

I think you might only want to do this for debugging. But for a general productive setup, I would assume that you rather have a file where you log ALL thing status changes, just like we currently log all item events.
My point is that it rather adds inhomogenous logging, if every binding decides for itself, how a status change is logged (message format, severity level etc.) It would be much nicer to have that done at a central place. This does not mean that you would not be able to tell that a log entry is related to a hue device of course.